src/librustdoc/clean/types.rs

```diff
@@ -86,22 +86,7 @@ impl FakeDefId {
     }
 
     #[inline]
-    crate fn as_local(self) -> Option<LocalDefId> {
-        match self {
-            FakeDefId::Real(id) => id.as_local(),
-            FakeDefId::Fake(idx, krate) => {
-                (krate == LOCAL_CRATE).then(|| LocalDefId { local_def_index: idx })
-            }
-        }
-    }
-
-    #[inline]
-    crate fn expect_local(self) -> LocalDefId {
-        self.as_local()
-            .unwrap_or_else(|| panic!("FakeDefId::expect_local: `{:?}` isn't local", self))
-    }
-
-    #[inline]
+    #[track_caller]
```

Comment:
Note: I did not remove `#[inline]`. It's just a quirk of the diff that it looks that way.